pkgs/tools/X11/imwheel/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, libX11, libXext, libXi, libXmu, libXt, libXtst }:
+
+stdenv.mkDerivation rec {
+  name = "imwheel-1.0.0pre12";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/imwheel/${name}.tar.gz";
```

```suggestion
    url = "mirror://sourceforge/imwheel/${name}.tar.gz";
```

fetchurl has mirror urls. See https://github.com/NixOS/nixpkgs/blob/04719a005202a9b0a8e815c90f94a5e9a1fce05d/pkgs/build-support/fetchurl/mirrors.nix#L8